### PR TITLE
Fix #245

### LINKED ISSUE #245: methods are not allowed for skipped fields when using multi-line doc comments
Title. Structopt 0.3.0 errors out with `methods are not allowed for skipped fields` if skipped fields have doc comments, but only multi-line ones.

Repro:

```rust
#[derive(StructOpt)]
struct Example {
    /// This does an internal thing.
    /// Run-on sentence explanation.
    #[structopt(skip)]
    works: String,

    /// This does another internal thing.
    ///
    /// # IMPORTANT NOTE
    #[structopt(skip)]
    fails: String,
}
```

### PR DESCRIPTION
**Pull request by CreepySkeleton**
Closes #245

### CODE REVIEW

**Comment by CreepySkeleton** [comment]
This is just a fast patch. I'm working on a big refactoring that introduces explicit AST and separates input processing into phases: `parse`, `validate`, `generate`. Current implementation is almost unworkable when it comes to parent - children relationships

File: tests/skip.rs [hunk 1]
```diff
@@ -92,3 +92,34 @@ fn skip_enum() {
...
+    #[structopt(name = "a")]
+    pub struct Opt {
+        #[structopt(skip, help = "internal_stuff")]
+        a: u32,
+
+        #[structopt(skip, long_help = "internal_stuff\ndo not touch")]
+        b: u32,
+
+        /// Not meant to be used by clap.
+        /// I want a default here.
```
Comment (TeXitoi) [inline tests/skip.rs]: Did you check the test fail without the fix? Because I think an empty line is needed at the second line to generate a long_help

Comment (CreepySkeleton) [reply tests/skip.rs]: Yep, forgot to change

**Comment by TeXitoi** [comment]
Thanks

**Pull request merged**